packages/orbit-components/src/Loading/index.tsx

```diff
@@ -1,191 +1,70 @@
-  theme: defaultTheme,
-};
-
-export const StyledSpinner = styled.svg<{ customSize?: Props["customSize"] }>`
-  ${({ customSize }) => css`
-    width: ${customSize ? `${customSize}px` : "40px"};
-    height: ${customSize ? `${customSize}px` : "40px"};
-    animation: ${SpinnerAnimation} 0.75s linear infinite;
-  `};
-`;
-
-const StyledSpinnerCircle = styled.circle<{ customSize?: Props["customSize"] }>`
-  ${({ type, theme, customSize }) => css`
-    fill: transparent;
-    stroke: ${type === TYPE_OPTIONS.BUTTON_LOADER ? "currentColor" : theme.orbit.paletteCloudDark};
-    stroke-width: 3px;
-    stroke-linecap: round;
-    stroke-dasharray: ${customSize ? `${customSize * 3 + 8}px` : "128px"};
-    stroke-dashoffset: ${customSize ? `${customSize + 24}px` : "64px"};
-  `};
-`;
-
-StyledSpinnerCircle.defaultProps = {
-  theme: defaultTheme,
-};
-
-const StyledLoader = styled.div`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-`;
-
-const StyledLoaderCircle = styled.div`
-  width: 8px;
-  height: 8px;
-  border-radius: 50%;
-  margin-${right}: 6px;
-  background: ${({ theme }) => theme.orbit.paletteCloudDark};
-  animation: ${LoaderAnimation} 1.25s infinite ease-in-out;
-  &:nth-child(2) {
-    animation-delay: 0.1s;
-  }
-  &:nth-child(3) {
-    animation-delay: 0.2s;
-    margin: 0;
-  }
-`;
-
-StyledLoaderCircle.defaultProps = {
-  theme: defaultTheme,
-};
+import useTheme from "../hooks/useTheme";
 
 const Loader = ({ type, customSize }) => {
+  const theme = useTheme();
   const isCircledIcon =
     type === TYPE_OPTIONS.BOX_LOADER ||
     type === TYPE_OPTIONS.SEARCH_LOADER ||
     type === TYPE_OPTIONS.INLINE_LOADER;
 
   if (customSize) {
     return (
-      <StyledSpinner viewBox={`0 0 ${customSize} ${customSize}`} customSize={customSize}>
-        <StyledSpinnerCircle
+      <svg
+        viewBox={`0 0 ${customSize} ${customSize}`}
+        className="orbit-loading-spinner animate-spinner h-xxl w-xxl"
+      >
+        <circle
           cx="50%"
           cy="50%"
-          r={customSize ? customSize / 2 - 2 : 18}
-          customSize={customSize}
+          fill="transparent"
+          strokeWidth={3}
+          stroke={type === TYPE_OPTIONS.BUTTON_LOADER ? "current" : theme.orbit.paletteCloudDark}
+          strokeLinecap="round"
+          strokeDasharray={`${customSize * 3 + 8}px`}
+          strokeDashoffset={`${Number(customSize) + 24}px`}
+          r={customSize / 2 - 2}
         />
-      </StyledSpinner>
+      </svg>
     );
   }
 
-  if (isCircledIcon)
+  if (isCircledIcon) {
     return (
-      <StyledLoader>
-        <StyledLoaderCircle />
-        <StyledLoaderCircle />
-        <StyledLoaderCircle />
-      </StyledLoader>
+      <div className="flex items-center justify-center">
+        <div className="animate-loader rounded-circle bg-cloud-dark h-xs w-xs me-[6px]" />
+        <div
+          className="animate-loader rounded-circle bg-cloud-dark h-xs w-xs me-[6px]"
```

Comment:
I see this `animate-loader` class applies `loader 1s 1.25s infinite ease-in-out` but it used to be `loader 1.25s infinite ease-in-out`. This must be changed on the tailwind-preset package.